guide/getting-started/11-common-user-flows.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: guide
+title: X Common user flows
+nav_order: 11
+parent: Getting started
+permalink: /getting-started/common-user-flows/
+---
+
+# Common user flows
+
+<!--
+
+## Software download & installation
+This step is many times overlooked, but before anyone uses your product, they first need to find it, get interested in it and choose to install it.
+
+## Software onboarding
+Bitcoin is complex, and so it is recommended to think through and carefully shape the first experience users have in your product. Without being overwhelming or getting into too much detail, this user flow should explain core concepts and features that allow users to create mental models on how they will use the application.
+
+Onboarding may be purely informational based on the content, but many times it also includes an initial setup that helps personalize the software towards the users specific needs and context.
+
+## Creating a new wallet
+I is important here to distinguish between wallet software, and the wallet that represents the users "account". This particular experience may be almost invisible or highly elaborate. While some mobile wallet applications choose to automatically create and secure a wallet during first launch, other wallets provide extensive configuration wizards.
+
+## Importing an existing wallet
+
+
+## Securing a wallet
+
+## Sending bitcoin
+
+## Requesting bitcoin
+
+## Wallet maintenance
+
+## Resolving a problem
+This can be a tricky experience to address. For one, non-custodial cryptocurrency management by nature places a lot of responsibility on the user. This also puts much of the burden of solving problems on users. The second aspect is that open-source software typically relies on online documentation and forums for "customer service". Ideally, software helps prevent errors as much as possible through techniques like validating user input and requiring extra approvals for impactful actions. When this is not possible though, and errors do occur, they should be communicated as clearly as possible.
```

Comment:
I think applications should really consider "customer support" as part of the safety net for resolving users problems. Although an agent may not be able to remedy things that happen on the network it can be helpful for someone to explain what has happened when an error or something unexpected occurs

Reply:
For sure. It's a pretty unthankful and often neglected job to write proper error messages and help sections, but it's super helpful for those tricky situations when something goes wrong.